src/Client/app/(admin)/invite-candidates/page.tsx

```
@@ -2,374 +1,457 @@
+                            onChange={(e) => setEditedEmail(e.target.value)}
+                            onBlur={() => {
+                                handleEditEmail();
+                                setEditingEmail(null);
+                            }}
+                            onKeyDown={(e) => {
+                                if (e.key === "Enter") {
+                                    handleEditEmail();
+                                    setEditingEmail(null);
+                                }
+                            }}
+                            autoFocus
```

Biome (1.9.4)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)